change/@fluentui-tokens-6453c666-41f3-4fb9-bba7-ca79fa23508d.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=32356&commitId=c92108ef3cebbbf3e76fe4a7bc1c7e543a78cb84&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.Full Overlay Dark Mode.chromium.png|974| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>